src/applications/vaos/components/layout/InPersonLayout.jsx

```diff
@@ -64,24 +68,43 @@ export default function InPersonLayout({ data: appointment }) {
             </div>
           )}
       </When>
-      <What>{typeOfCareName || 'Type of care information not available'}</What>
+      <What>{typeOfCareName}</What>
       {oracleHealthProviderName && <Who>{oracleHealthProviderName}</Who>}
       <Where
         heading={
           APPOINTMENT_STATUS.booked === status ? 'Where to attend' : undefined
         }
       >
-        {!!facility === false && (
-          <>
-            <span>Facility details not available</span>
-            <br />
-            <NewTabAnchor href="/find-locations">
-              Find facility information
-            </NewTabAnchor>
-            <br />
-            <br />
-          </>
-        )}
+        {/* When the services return a null value for the facility (no facility ID) for all appointment types */}
+        {!!facility === false &&
+          !!facilityId === false && (
+            <>
+              <span>Facility details not available</span>
+              <br />
+              <NewTabAnchor href="/find-locations">
+                Find facility information
+              </NewTabAnchor>
+              <br />
+              <br />
+            </>
+          )}
+        {/* When the services return a null value for the facility (but receive the facility ID) */}
```

Comment:
In the spec it says:

<img width="337" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/2030323/d0be499f-bfbe-4fce-af67-7da02065c0ee">

Is this the facility name something we can retrieve using just the ID?

Reply:
Current behavior (on old design) doesn't display the name and I cannot find anywhere that would exist if we only have the id. My assumption is that we wouldn't be getting that information back from the facilities endpoint but would be able to construct the link with the id to allow users to get more information about that facility.

Reply:
That's fair enough, but might want to check with @Cieramaddox to make sure.